Assets/Prefabs/Robot.prefab

```diff
@@ -358,7 +358,7 @@ MonoBehaviour:
     creationDelayMs: 0
     laserDevice: /dev/tty_in1
     motorPort: outC
-    laserDutyCycle: -44
+    laserDutyCycle: 44
```

Comment:
It's the duty cycle that corresponds to around 270 rpm when battery is charged.

LIDAR works with speed between 200 and 300 something.

44 instead of -44 because we are stickicking to the "official" sheme we made on ev3dev.org, interfacing the XV11Lidar